plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -247,6 +247,27 @@ fn run_cmd(cmd: &str, args: &str) -> Result<ExitStatus, InternalError> {
     Ok(status)
 }
 
+fn get_name_and_version(line: &str) -> (&str, &str) {
```

Comment:
An example of the expected input format as a function comment would have been nice.